### PR TITLE
[SWA-30][FEAT] - Add $ARB token to the SDK

### PR DESCRIPTION
**Pull request by ElRodrigote**
## Fixes: [SWA-30](https://linear.app/swaprdev/issue/SWA-30/add-arb-token-to-common-tokens)

# Description

* Add $ARB to Mainnet, Arbitrum One and Arbitrum Goerli

### CODE REVIEW

File: src/entities/token.ts [hunk 1]
```diff
@@ -11,6 +11,27 @@ export class Token extends Currency {
...
+      'DXdao'
+    ),
+    [ChainId.ARBITRUM_ONE]: new Token(
+      ChainId.ARBITRUM_ONE,
+      '0x912CE59144191C1204E64559FE8253a0e49E6548',
+      18,
+      'ARB',
+      'DXdao'
+    ),
+    [ChainId.MAINNET]: new Token(ChainId.MAINNET, '0xB50721BCf8d664c30412Cfbc6cf7a15145234ad1', 18, 'ARB', 'DXdao'),
```
Comment (berteotti) [inline src/entities/token.ts]: Token name should be `Arbitrum`

**Review by berteotti — CHANGES_REQUESTED** [review]
Code looks good, just requested a small change

File: src/entities/token.ts (hunk 1 above)
Comment (ElRodrigote) [reply src/entities/token.ts]: Shit you're right, I thought I changed it, sorry! ON it!

**Review by berteotti — APPROVED** [review]
This has been tested on the preview and it works. You can remove dist folder and merge. Good job Rorry!

**Pull request merged**